cmake/Buildevent-driven.cmake

```
@@ -5,10 +5,12 @@ include(YCMEPHelper)
 include(FindOrBuildPackage)
 
 find_or_build_package(YARP QUIET)
+find_or_build_package(YCM QUIET)
 
 ycm_ep_helper(event-driven TYPE GIT
               STYLE GITHUB
               REPOSITORY robotology/event-driven.git
-              DEPENDS YARP
+              DEPENDS	YARP
+              	      	YCM
```

Hi @francodipietro , it is a good idea to never mix tabs and spaces, otherwise the result can depend on the tab size used by the viewer. 
```suggestion
                 YCM
```

---

Sure. My bad.